cinch/library/jenkins_user_api.py

```diff
@@ -0,0 +1,49 @@
+from ansible.module_utils.basic import AnsibleModule
+from tempfile import NamedTemporaryFile
+import subprocess
+import os
+
+
+MYFILE = """
+import hudson.model.User;
+User u = User.get("{0}");
+def p = u.getProperty(jenkins.security.ApiTokenProperty.class);
+println p.getApiToken();
+"""
+
+
+def main():
+    module = AnsibleModule(
+        argument_spec={
+            'user': {'required': True, 'type': 'str'},
+            'cli_jar': {'default':
+                        '/var/cache/jenkins/war/WEB-INF/jenkins-cli.jar'},
+            'jenkins_url': {'default': 'http://localhost:8080'}
+        }
+    )
+    args = type('Args', (object,), module.params)
+    groovy = NamedTemporaryFile(delete=False)
+    groovy.write(MYFILE.format(args.user))
+    groovy.close()
+    process = ['/usr/bin/java',
```

Comment:
Probably unlikely, but we might want to set this as a module argument in the event that the java binary is stored elsewhere.